app/views/decidim/consultations/consultation_summaries/_question.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="card consultations-card">
+  <div class="row collapse card--process__row">
+    <div class="columns mediumlarge-8 large-9 card--process__column">
+      <div class="card__content">
+        <%= link_to "#{question_path(question)}/summary", class: "card__link" do %>
+          <h2 class="heading5"><%= decidim_sanitize translated_attribute question.title %></h2>
+        <% end %>
+        <p><%= translated_attribute question.subtitle %></p>
+      </div>
+    </div>
+
+    <div class="columns mediumlarge-8 large-3 card--process__column">
+      <div class="card--full__image consultations-card__image"
+           style="background-image: url(<%= question.banner_image_url %>)">
+        <div class="card__content row collapse">
+          <div class="columns large-10 large-push-1">
+            <%= link_to "#{question_path(question)}/summary",
```

Comment:
This should be done by using Rails routes to take into account query variables (ie: locale)
```suggestion
      <%= link_to "#{Rails.application.routes.url_helpers.question_summary_path(question, locale: params[:locale])}",
```